foxglove/cmd/config.go

```diff
@@ -0,0 +1,161 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func newConfigCommand() *cobra.Command {
+	configCmd := &cobra.Command{
+		Use:   "config",
+		Short: "Manage CLI configuration",
+		Long: `Manage CLI configuration values.
+Available configuration keys:
+  - project-id: Default project ID for commands
+  - api-key: API key for authentication`,
+	}
+
+	configCmd.AddCommand(newConfigGetCommand())
+	configCmd.AddCommand(newConfigSetCommand())
+	configCmd.AddCommand(newConfigUnsetCommand())
+
+	return configCmd
+}
+
+func newConfigGetCommand() *cobra.Command {
+	getCmd := &cobra.Command{
+		Use:   "get [KEY]",
+		Short: "Get a configuration value",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			key := args[0]
+
+			if !isValidConfigKey(key) {
+				dief("Invalid configuration key '%s'. Valid keys are: project-id, api-key", key)
```

Comment:
Can we read the valid keys from a variable? Seems like this will get out of date.